src/roman_datamodels/stnode/_tagged.py

```diff
@@ -45,7 +43,108 @@ def name_from_tag_uri(tag_uri):
     return tag_uri_split
 
 
-class TaggedObjectNode(DNode):
+class _TaggedNodeMixin(ABC):
```

Comment:
All the tagged objects share all of these and had copies of them so I pulled them out into this.